src/api/auth.ts

```
@@ -0,0 +1,48 @@
+
+export const signup = async (userData: {
+  fullName: string;
+  companyName: string;
+  email: string;
+  password: string;
+}) => {
+  try {
+    const response = await api.post("/auth/register", userData);
+    return response.data;
+  } catch (error: any) {
+    if (error.response && error.response.data && error.response.data.message) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)